src/olympia/reviewers/views.py

```diff
@@ -933,8 +938,9 @@ def review(request, addon, channel=None):
     except Version.DoesNotExist:
         show_diff = None
 
-    # The actions we should show a minimal form for.
-    actions_minimal = [k for (k, a) in actions if not a.get('minimal')]
+    # The actions we shouldn't show a minimal form for.
+    actions_full = [
```

Comment:
note: I didn't actually change the behaviour here - `actions_minimal` was misnamed.